kahypar/partition/preprocessing/large_he_remover.h

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * This file is part of KaHyPar.
+ *
+ * Copyright (C) 2022 Tobias Heuer <tobias.heuer@kit.edu>
+ *
+ * KaHyPar is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * KaHyPar is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with KaHyPar.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ ******************************************************************************/
+
+
+#pragma once
+
+#include "kahypar/definitions.h"
+#include "kahypar/partition/context.h"
+
+namespace kahypar {
+
+class LargeHyperedgeRemover {
```

Comment:
Could you add a small test case for this class? 🙂

Reply:
ungern, but done